oct/utilities/run.py

```diff
@@ -58,7 +58,7 @@ def run(args):
         del kwargs['func']
 
     project_path = kwargs.pop('project_path')
-    config = configure(project_path)
+    config = configure(project_path, kwargs['config_file'])
```

Comment:
If you want to keep then you will need to update `test_hq.py` file in tests directory.
Can still raise error if run called without using argparse